pysat/utils/_core.py

```diff
@@ -37,19 +37,6 @@ def set_data_dir(path=None, store=True):
         raise ValueError('Path %s does not lead to a valid directory.' % path)
```

Comment:
Recommend updating the style of this error message formatting.

Reply:
using `string.format()`?

Reply:
Yes, but without importing string: `'I am text {:s}'.format('and more text')`

Reply:
fixed